<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SyntaxTutor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/syntaxtutor.png</normaloff>:/resources/syntaxtutor.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: #1F1F1F;
color: #E0E0E0;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>432</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>LL(1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>432</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SLR(1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tutorial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>432</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #2F2F2F;        /* Gris muy oscuro, casi al tono de fondo */
    color:            #CCCCCC;        /* Gris claro, sin llegar a blanco puro */
    border:           1px solid #444; /* Borde sutil para definir límites */
    padding:          8px 16px;
    border-radius:    4px;
    font-size:        14px;
    font-family:      'Noto Sans';
}
QPushButton:hover {
    background-color: #3A3A3A;        /* Un poco más claro al pasar el cursor */
    color:            #FFFFFF;
}
QPushButton:pressed {
    background-color: #262626;        /* Oscurece un pelín al pulsar */
}</string>
        </property>
        <property name="text">
         <string>Tutorial</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>17</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QRadioButton" name="lv1Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    color:    #DDDDDD;
    font-size:16px;
    spacing:  6px;
}
QRadioButton::indicator {
    width:        16px;
    height:       16px;
    border-radius:8px;
    border:       2px solid #555555;
    background:   transparent;
}
QRadioButton::indicator:checked {
    background: #00ADB5;
    border:     2px solid #00ADB5;
}
QRadioButton::indicator:checked + QLabel {
    color: #00ADB5;
}
</string>
        </property>
        <property name="text">
         <string>Nivel 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QRadioButton" name="lv2Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    color:    #DDDDDD;
    font-size:16px;
    spacing:  6px;
}
QRadioButton::indicator {
    width:        16px;
    height:       16px;
    border-radius:8px;
    border:       2px solid #555555;
    background:   transparent;
}
QRadioButton::indicator:checked {
    background: #00ADB5;
    border:     2px solid #00ADB5;
}
QRadioButton::indicator:checked + QLabel {
    color: #00ADB5;
}
</string>
        </property>
        <property name="text">
         <string>Nivel 2</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QRadioButton" name="lv3Button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    color:    #DDDDDD;
    font-size:16px;
    spacing:  6px;
}
QRadioButton::indicator {
    width:        16px;
    height:       16px;
    border-radius:8px;
    border:       2px solid #555555;
    background:   transparent;
}
QRadioButton::indicator:checked {
    background: #00ADB5;
    border:     2px solid #00ADB5;
}
QRadioButton::indicator:checked + QLabel {
    color: #00ADB5;
}
</string>
        </property>
        <property name="text">
         <string>Nivel 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAcercaDe">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionSobre_la_aplicaci_n"/>
    <addaction name="actionReferencia_LL_1"/>
    <addaction name="actionReferencia_SLR_1"/>
   </widget>
   <addaction name="menuAcercaDe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobre_la_aplicaci_n">
   <property name="text">
    <string>Sobre la aplicación</string>
   </property>
  </action>
  <action name="actionReferencia_LL_1">
   <property name="text">
    <string>Referencia LL(1)</string>
   </property>
  </action>
  <action name="actionReferencia_SLR_1">
   <property name="text">
    <string>Referencia SLR(1)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
